guides/doc-Upgrading_and_Updating/topics/updating_satellite_server_to_next_minor_version.adoc

```diff
@@ -88,35 +88,35 @@ sslclientkey = /etc/pki/katello/certs/org-debug-cert.pem
 sslcacert = /etc/pki/katello/certs/katello-server-ca.crt
 sslverify = 1
 
-[{RepoRHEL7Server}]
-name=Red Hat Enterprise Linux 7 Server RPMs x86_64
-baseurl=https://{foreman-example-com}/pulp/content/My_Organization/Library/content/dist/rhel/server/7/7Server/x86_64/os/
+[{RepoRHEL8BaseOS}]
+name=Red Hat Enterprise Linux 8 for x86_64 - BaseOS (RPMs)
+baseurl=https://{foreman-example-com}/pulp/content/My_Organization/Library/content/dist/rhel8/8/x86_64/baseos/os
 enabled=1
 sslclientcert = /etc/pki/katello/certs/org-debug-cert.pem
 sslclientkey = /etc/pki/katello/certs/org-debug-cert.pem
 sslcacert = /etc/pki/katello/certs/katello-server-ca.crt
 sslverify = 1
 
-[{RepoRHEL7ServerSatelliteServerProductVersion}]
-name={ProjectNameX} for RHEL 7 Server RPMs x86_64
-baseurl=https://{foreman-example-com}/pulp/content/My_Organization/Library/content/dist/rhel/server/7/7Server/x86_64/satellite/{ProjectVersion}/os/
+[{RepoRHEL8ServerSatelliteServerProductVersion}]
+name={ProjectName} {ProjectVersion} for RHEL 8 RPMs x86_64
+baseurl=https://{foreman-example-com}/pulp/content/My_Organization/Library/content/dist/layered/rhel8/x86_64/satellite/{ProjectVersion}/os
 enabled=1
 sslclientcert = /etc/pki/katello/certs/org-debug-cert.pem
 sslclientkey = /etc/pki/katello/certs/org-debug-cert.pem
 sslcacert = /etc/pki/katello/certs/katello-server-ca.crt
 
-[{RepoRHEL7ServerSatelliteMaintenanceProductVersion}]
-name={ProjectName} Maintenance 6 for RHEL 7 Server RPMs x86_64
-baseurl=https://{foreman-example-com}/pulp/content/My_Organization/Library/content/dist/rhel/server/7/7Server/x86_64/sat-maintenance/6/os/
+[{RepoRHEL8ServerSatelliteMaintenanceProductVersion}]
+name={ProjectName} Maintenance {ProjectVersion} for RHEL 8 RPMs x86_64
+baseurl=https://{foreman-example-com}/pulp/content/My_Organization/Library/content/dist/layered/rhel8/x86_64/sat-maintenance/{ProjectVersion}/os
 enabled=1
 sslclientcert = /etc/pki/katello/certs/org-debug-cert.pem
 sslclientkey = /etc/pki/katello/certs/org-debug-cert.pem
 sslcacert = /etc/pki/katello/certs/katello-server-ca.crt
 sslverify = 1
 
-[{RepoRHEL7ServerSoftwareCollections}]
-name=Red Hat Software Collections RPMs for Red Hat Enterprise Linux 7 Server x86_64
-baseurl=https://{foreman-example-com}/pulp/content/My_Organization/Library/content/dist/rhel/server/7/7Server/x86_64/rhscl/1/os/
+[{RepoRHEL8AppStream}]
```

Comment:
same as above, I'd put the appstream section just after baseos, to keep them together